HW8/Game/src/Poison.java

```diff
@@ -0,0 +1,3 @@
+public class Poison {
```

Comment:
It's a good one, I see you were thinking about adding some poisons but then decided to submit with sword and shield only :)